### PR TITLE
ENH: kwarg format for `print(inst)`

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

The current output of kwargs can be difficult to read, especially when multiple kwargs which include lengthy strings are used.  This makes it particularly difficult when comparing multiple instruments with different TLEs when using pysatMissions.

This parses each kwarg as a separate line when running `print(inst)`.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

```
import datetime as dt
import numpy as np
import pandas as pds
import pysat

from pysatMissions.instruments import missions_sgp4

# Use SPORT TLEs as an example
line1 = '1 55129U 98067UW  23039.82653086  .00049915  00000-0  78305-3 0  9991'
line2 = '2 55129  51.6397 243.8305 0001856 340.7022  19.3898 15.53177014  6499'


file_date_range = pds.date_range(dt.datetime(2022, 12, 30),
                                 dt.datetime(2023, 2, 28))

sport = pysat.Instrument(inst_module=missions_sgp4, tle1=line1, tle2=line2,
                         use_header=True, file_date_range=file_date_range)

print(sport)
```
in develop:
```
pysat Instrument object
-----------------------
Platform: 'missions'
Name: 'sgp4'
Tag: ''
Instrument id: ''

Data Processing
---------------
Cleaning Level: 'clean'
Data Padding: None
Keyword Arguments Passed to list_files: {'file_date_range': DatetimeIndex(['2022-12-30', '2022-12-31', '2023-01-01', '2023-01-02',
               '2023-01-03', '2023-01-04', '2023-01-05', '2023-01-06',
               '2023-01-07', '2023-01-08', '2023-01-09', '2023-01-10',
               '2023-01-11', '2023-01-12', '2023-01-13', '2023-01-14',
               '2023-01-15', '2023-01-16', '2023-01-17', '2023-01-18',
               '2023-01-19', '2023-01-20', '2023-01-21', '2023-01-22',
               '2023-01-23', '2023-01-24', '2023-01-25', '2023-01-26',
               '2023-01-27', '2023-01-28', '2023-01-29', '2023-01-30',
               '2023-01-31', '2023-02-01', '2023-02-02', '2023-02-03',
               '2023-02-04', '2023-02-05', '2023-02-06', '2023-02-07',
               '2023-02-08', '2023-02-09', '2023-02-10', '2023-02-11',
               '2023-02-12', '2023-02-13', '2023-02-14', '2023-02-15',
               '2023-02-16', '2023-02-17', '2023-02-18', '2023-02-19',
               '2023-02-20', '2023-02-21', '2023-02-22', '2023-02-23',
               '2023-02-24', '2023-02-25', '2023-02-26', '2023-02-27',
               '2023-02-28'],
              dtype='datetime64[ns]', freq='D')}
Keyword Arguments Passed to load: {'tle1': '1 55129U 98067UW  23039.82653086  .00049915  00000-0  78305-3 0  9991', 'tle2': '2 55129  51.6397 243.8305 0001856 340.7022  19.3898 15.53177014  6499', 'use_header': True, 'epoch': Timestamp('2022-12-30 00:00:00', freq='D')}
Keyword Arguments Passed to preprocess: {}
Keyword Arguments Passed to download: {}
Keyword Arguments Passed to list_remote_files: {}
Keyword Arguments Passed to clean: {}
Keyword Arguments Passed to init: {}
Custom Functions: 0 applied

Local File Statistics
---------------------
Number of files: 61
Date Range: 30 December 2022 --- 28 February 2023

Loaded Data Statistics
----------------------
No loaded data.

```

This branch:
```
pysat Instrument object
-----------------------
Platform: 'missions'
Name: 'sgp4'
Tag: ''
Instrument id: ''

Data Processing
---------------
Cleaning Level: 'clean'
Data Padding: None
Keyword Arguments Passed to list_files: 
    'file_date_range': DatetimeIndex(['2022-12-30', '2022-12-31', '2023-01-01', '2023-01-02',
               '2023-01-03', '2023-01-04', '2023-01-05', '2023-01-06',
               '2023-01-07', '2023-01-08', '2023-01-09', '2023-01-10',
               '2023-01-11', '2023-01-12', '2023-01-13', '2023-01-14',
               '2023-01-15', '2023-01-16', '2023-01-17', '2023-01-18',
               '2023-01-19', '2023-01-20', '2023-01-21', '2023-01-22',
               '2023-01-23', '2023-01-24', '2023-01-25', '2023-01-26',
               '2023-01-27', '2023-01-28', '2023-01-29', '2023-01-30',
               '2023-01-31', '2023-02-01', '2023-02-02', '2023-02-03',
               '2023-02-04', '2023-02-05', '2023-02-06', '2023-02-07',
               '2023-02-08', '2023-02-09', '2023-02-10', '2023-02-11',
               '2023-02-12', '2023-02-13', '2023-02-14', '2023-02-15',
               '2023-02-16', '2023-02-17', '2023-02-18', '2023-02-19',
               '2023-02-20', '2023-02-21', '2023-02-22', '2023-02-23',
               '2023-02-24', '2023-02-25', '2023-02-26', '2023-02-27',
               '2023-02-28'],
              dtype='datetime64[ns]', freq='D')
Keyword Arguments Passed to load: 
    'tle1': 1 55129U 98067UW  23039.82653086  .00049915  00000-0  78305-3 0  9991
    'tle2': 2 55129  51.6397 243.8305 0001856 340.7022  19.3898 15.53177014  6499
    'use_header': True
    'epoch': 2022-12-30 00:00:00
Keyword Arguments Passed to preprocess: 
Keyword Arguments Passed to download: 
Keyword Arguments Passed to list_remote_files: 
Keyword Arguments Passed to clean: 
Keyword Arguments Passed to init: 
Custom Functions: 0 applied

Local File Statistics
---------------------
Number of files: 61
Date Range: 30 December 2022 --- 28 February 2023

Loaded Data Statistics
----------------------
No loaded data.
```

**Test Configuration**:
* Operating system: Monterrey
* Version number: Python 3.10.8

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by aburrell** [comment]
Recommend changing to be something like:
```
Keyword Arguments Passed:
To load: 
    'tle1': 1 55129U 98067UW  23039.82653086  .00049915  00000-0  78305-3 0  9991
    'tle2': 2 55129  51.6397 243.8305 0001856 340.7022  19.3898 15.53177014  6499
    'use_header': True
    'epoch': 2022-12-30 00:00:00
To init: 
    'user': 'Otterly'
    'password': 'otterly@adorable.edu'
```

So that the output isn't very long with data that isn't important

**Comment by jklenzing** [comment]
> 

Should I drop the ones without kwargs then?

**Comment by aburrell** [comment]
Yes, exactly.

File: pysat/_instrument.py [hunk 1]
```diff
@@ -682,9 +682,14 @@ def __str__(self):
...
+        output_str += 'Keyword Arguments Passed: \n'
         for routine in self.kwargs.keys():
-            output_str += 'Keyword Arguments Passed to {:s}: '.format(routine)
-            output_str += "{:s}\n".format(self.kwargs[routine].__str__())
+            # Only output for routine if kwargs are present.
+            if len(self.kwargs[routine].keys()) > 0:
+                output_str += "  To {:s}: \n".format(routine)
+                for key in self.kwargs[routine].keys():
+                    output_str += "    '{:s}': {:s}\n".format(
+                        key, str(self.kwargs[routine][key]))
```
Comment (jklenzing) [inline pysat/_instrument.py:692]:
```suggestion
                # Only output for routine if kwargs are present.
                if len(self.kwargs[routine].keys()) > 0:
                    output_str += "  To {:s}: \n".format(routine)
                    for key in self.kwargs[routine].keys():
                        output_str += "    '{:s}': {:s}\n".format(
                            key, str(self.kwargs[routine][key]))
```

**Review by aburrell — APPROVED** [review]

**Pull request merged**